.github/workflows/ci.yaml

```diff
@@ -59,7 +59,7 @@ jobs:
     - name: Setup node
       uses: actions/setup-node@v3
       with:
-        node-version: "14"
+        node-version: "18"
```

Comment:
We need to confirm the nightly and release workflows will still work. I do not see where the node version is set in those two workflows. So I do not know how it knows which node version to use.

Reply:
Actually it does not need the setup node in any github workflow, builds are done by podman container with image `registry.access.redhat.com/ubi8/nodejs-18`(https://github.com/kiali/openshift-servicemesh-plugin/blob/f49165ccdcafd04cff81f88d4166f1a1c50cf327/plugin/Dockerfile#L1)